doc/source/cluster/kuberay.md

```diff
@@ -0,0 +1,96 @@
+# Deploying with Kuberay (experimental)
+
+```{admonition} What is Kuberay?
+[Kuberay](https://github.com/ray-project/kuberay) is a set of tools for running Ray on Kubernetes.
+It has been used by some larger corporations to deploy Ray on their infrastructure.
+Going forward, we would like to make this way of deployment accessible and seamless for
+all Ray users and standardize Ray deployment on Kubernetes around Kuberay's operator.
+Presently you should consider this integration a minimal viable product that is not polished
```

Comment:
I think we should label this integration as "Experimental" or "Alpha" to be consistent about setting expectations for other Ray features

Reply:
The header does says "Experimental." Wouldn't hurt to repeat that word in this sentence though.